prov/util/src/util_cntr.c

```diff
@@ -204,11 +204,55 @@ static struct fi_ops_cntr util_cntr_no_wait_ops = {
 	.wait = fi_no_cntr_wait,
 };
 
+static void util_peer_cntr_inc(struct fid_peer_cntr *cntr)
+{
+	struct util_cntr *util_cntr = cntr->fid.context;
+
+	util_cntr->cntr_fid.ops->add(&util_cntr->cntr_fid, 1);
+}
+
+static void util_peer_cntr_incerr(struct fid_peer_cntr *cntr)
+{
+	struct util_cntr *util_cntr = cntr->fid.context;
+
+	util_cntr->cntr_fid.ops->adderr(&util_cntr->cntr_fid, 1);
+}
+
+static struct fi_ops_cntr_owner util_peer_cntr_owner_ops = {
+	.size = sizeof(struct fi_ops_cntr_owner),
+	.inc = &util_peer_cntr_inc,
+	.incerr = &util_peer_cntr_incerr,
+};
+
+static uint64_t ofi_peer_cntr_read(struct fid_cntr *cntr_fid)
```

Comment:
Not sure about the return type - I think it should be either size_t or int.

Reply:
It's the change in https://github.com/ofiwg/libfabric/pull/9117. I put it here just for making the PR work.

For the return type, it needs to be consistent with fi_cntr_read, which returns uint64_t